app/support/styles.scss

```diff
@@ -1,6 +1,15 @@
+/* stylelint-disable selector-class-pattern */
+
+.Support {
+    a:not(.btn),
+    .btn-link {
```

Comment:
Granted, this sort of logic doesn't suit itself to a BEM-y structure, but I think that's because it doesn't really make sense. Do we want all links on the whole site bold? If not, why just this page? The existing [support page](https://osf.io/support/) has normal links.

Reply:
If we do want all links on the whole site bold, that's fine, but that'd belong in an application-level stylesheet.

Reply:
It's specifically on this page to deal with the white background not contrasting well with the light blue links. If we make all links bold, that would probably be overkill. I'm willing to refactor later if we need this in multiple places.

Reply:
Blue link on white background is an astoundingly common pattern. Bold links would look fine, it's a reasonable choice.

It might be worth a product consult.  I think it's our responsibility to keep the design reasonably consistent as we parade through the whole site; I don't want us to leave a trail of special cases if we can avoid it.

Reply:
This change was the result of the product consult. Agree we should be consistent, and that would be a part of the refactor, if necessary. This may be superseded by the bootstrap 4 upgrade, in which case I'm fine with removing it.

Reply:
Agreed; much like the success and info button colors, anything that's going to have to happen everywhere anyways should just happen everywhere in our pages.

Reply:
I agree, but I'm not comfortable at this point asserting that it would happen everywhere

Reply:
I'm pretty comfortable with the default being bolder and accessible, and we can make changes if we hate it elsewhere, but I'm also checking with Product on if they are similarly comfortable. But yeah, if we do bootstrap 4, and it goes away with that, all the better.

Reply:
Product's cool with it, too. As long as it's limited to our pages, shouldn't require a lot of concern.